SharkCage/CageManager/CageManager.cpp

```diff
@@ -55,7 +55,19 @@ int main()
 		return 1;
 	}
 
-	if (cage_manager.ProcessRunning(cage_data.app_path) || (cage_data.hasAdditionalAppInfo() && cage_manager.ProcessRunning(cage_data.additional_app_path.value())))
+	if (!ValidateBinariesToLaunch(cage_data))
+	{
+		std::cout << "Validity check of binaries to launch failed." << std::endl;
+
+		std::wstring result_data;
+		network_manager.Send(sender, CageMessage::RESPONSE_FAILURE, L"Verification of the integrity for one or more of the process(es) you are"
+			" trying to start on the secure desktop has been failed. Please open the config in the Configurator.", result_data);
```

Comment:
typo: should be "has failed" instead of "has been failed"
Also, what does opening it in the configurator do / help to solve the issue? If the user just saves the config again without properly checking it he could save a compromised config. Maybe it would be better to say something like "[...]. This can happen if an app was updated between creating the configuration and opening it. Please take a look at the configuration in the CageConfigurator and (re)save it if you are sure everything is in order."